README.md

```diff
@@ -316,7 +316,7 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
    You can download this model with `ilab model download` command as well.
 
-4. The InstructLab CLI auto-detects your hardware and select the exact system profile that matches your machine. System profiles populate the `config.yaml` file with the proper parameter values based on your detected CPU/GPU types. This system is only applicable to Apple M-Series Chips, Nvidia GPUs, and Intel/AMD CPUs.
+4. The InstructLab Core auto-detects your hardware and select the exact system profile that matches your machine. System profiles populate the `config.yaml` file with the proper parameter values based on your detected CPU/GPU types. This system is only applicable to Apple M-Series Chips, Nvidia GPUs, and Intel/AMD CPUs.
```

Comment:
@nathan-weinberg is this technically accurate? Because I thought that it was the CLI that auto detects, but this might be more orchestration, so I might be wrong

Reply:
Seems better with `Core` as a function.

Reply:
updated to `InstructLab Core package`